numpy/core/fromnumeric.py

```diff
@@ -3125,10 +3125,35 @@ def around(a, decimals=0, out=None):
     -----
     For values exactly halfway between rounded decimal values, NumPy
     rounds to the nearest even value. Thus 1.5 and 2.5 round to 2.0,
-    -0.5 and 0.5 round to 0.0, etc. Results may also be surprising due
-    to the inexact representation of decimal fractions in the IEEE
-    floating point standard [1]_ and errors introduced when scaling
-    by powers of ten.
+    -0.5 and 0.5 round to 0.0, etc.
+
+    `np.around` uses a fast but sometimes inexact algorithm to round
```

Comment:
Probably want double backticks here, the single ticks would try to make a link, which seems redundant.

Reply:
This and some others in this paragraph appear in italics in the [rendered doc](https://9043-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/generated/numpy.around.html#numpy.around), they should have double backticks or be fixed to resolve to a link.